ledger/src/blockstore.rs

```diff
@@ -995,20 +968,8 @@ impl Blockstore {
                             reed_solomon_cache,
                         )
                         .ok(),
-                    ErasureMetaStatus::DataFull => {
-                        Self::submit_metrics(slot, erasure_meta, false, "complete".into(), 0);
-                        None
-                    }
-                    ErasureMetaStatus::StillNeed(needed) => {
-                        Self::submit_metrics(
-                            slot,
-                            erasure_meta,
-                            false,
-                            format!("still need: {needed}"),
-                            0,
-                        );
-                        None
-                    }
+                    ErasureMetaStatus::DataFull => None,
```

Comment:
nit can
```suggestion
                    ErasureMetaStatus::DataFull | ErasureMetaStatus::StillNeed(_) => None,
```

Reply:
Or even remove `ErasureMetaStatus` and replace with a boolean and simpler `ErasureMeta::status`

Reply:
yeah, I was going to remove `ErasureMetaStatus` in a follow up PR.
I can do it in this same PR. either way is fine with me.